packages/amplify_datastore/example/android/app/build.gradle

```diff
@@ -21,13 +16,23 @@ if (flutterVersionName == null) {
     flutterVersionName = '1.0'
 }
 
-apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
-apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
+// REMOVE IF TEST PASSES
+// apply plugin: 'com.android.application'
+// apply plugin: 'kotlin-android'
+// apply plugin: 'kotlin-parcelize'
+// apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
+
+plugins {
+    id "com.android.application"
+    id "kotlin-android"
+    id "dev.flutter.flutter-gradle-plugin"
+}
```

Comment:
Flutter docs say this block should be at the top of the doc. https://docs.flutter.dev/release/breaking-changes/flutter-gradle-plugin-apply#androidappbuildgradle